src/virtualenv/interpreters/create/cpython/cpython2.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import, unicode_literals
+
+import abc
+
+import six
+
+from virtualenv.util.path import Path, copy
+
+from .common import CPython, CPythonPosix, CPythonWindows
```

Comment:
mostly because I did not find a tool to automate it 😢 any suggestions? I'd prefer to make relative what's possible without (or at most 1 level ``..`` and keep everything else as absolute)